src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -29,15 +29,28 @@ public class PersonCard extends UiPart<Region> {
     @FXML
     private HBox cardPane;
     @FXML
-    private Label name;
-    @FXML
     private Label id;
     @FXML
+    private Label name;
+    @FXML
     private Label phone;
     @FXML
+    private Label email;
+    @FXML
     private Label address;
     @FXML
-    private Label email;
+    private Label schedule;
+    @FXML
+    private Label subject;
```

Comment:
The subject and schedule should be displayed as blue color tags like the UI.png
Can make use of FlowPane below

Reply:
Thank you for the suggestion. I have made the changes like what you suggested